pkg/k8s/secrets/transformers_test.go

```diff
@@ -1,16 +1,19 @@
 package secrets
 
 import (
+	"testing"
+
 	"github.com/SAP/stewardci-core/pkg/k8s/fake"
 	"gotest.tools/assert"
-	"testing"
+	v1 "k8s.io/api/core/v1"
 )
 
 func Test_UniqueNameTransformer(t *testing.T) {
-	secret := fake.Secret("name", "secret")
+	secret := fake.SecretWithType("name", "secret", v1.SecretTypeDockercfg)
 	result := UniqueNameTransformer()(secret)
 	assert.Equal(t, "", result.GetName())
-	assert.Equal(t, "name", result.GetGenerateName())
+	assert.Equal(t, "name-", result.GetGenerateName())
+	assert.Equal(t, v1.SecretTypeDockercfg, result.Type)
```

Comment:
Do we need this check? If yes, why checking only this but no other fields?

Reply:
If we set the type we should also check it IMHO.
Other fields are not set in the test.